TECHMANIA/Assets/Scripts/DiscordController.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using UnityEngine;
+
+public class DiscordController : MonoBehaviour
```

Comment:
This class doesn't need to inherit from `MonoBehaviour`, nor does it need to `using UnityEngine`.